src/components/AboutHome/AboutHome.js

```diff
@@ -49,6 +50,20 @@ const AboutHome = ({ user, setCurrentLink }) => {
     );
   }, []);
 
+  //Display login warnings for the login initial authorizer user with no responsibilities.
+  useEffect(() => {
+    //get the current user
+    const cdxUser = JSON.parse(localStorage.getItem("ecmps_user"));
+
+    //check if the user has initial authorizer role with no responsibilities
+    if (cdxUser?.roles?.includes(config.app.initialAuthorizerRole) && Object.keys(cdxUser?.facilities || {}).length === 0)
```

Comment:
@EvanSun220 The check for no responsibilities should be applied for any users that have the Sponsor, Submitter, Preparer, or Initial Authorizer roles.  

The example user given in the ticket only had the IA role, but the check should apply to all users that can prepare or submit data.